script/mysql-commands.sql

```diff
@@ -0,0 +1,15 @@
+DROP DATABASE IF EXISTS testQueue;
+CREATE DATABASE testQueue;
+USE testQueue;
+CREATE TABLE queue (
+    hash CHAR(64),
+    PRIMARY KEY(hash),
+    message VARCHAR(120) NOT NULL,
+    requeued BOOLEAN DEFAULT FALSE,
+    heartbeatDate DATETIME,
+    owner VARCHAR(256),
+    isProcessing BOOLEAN DEFAULT FALSE,
+    index isProcessing(isProcessing),
```

Comment:
Should be uppercase. https://tests-always-included.github.io/style-guide/lang/sql/
Looks like the SQL guide isn't linked up.  I'll go fix that.